### PR TITLE
Prometheus Integration Pack

### PR DESCRIPTION
**Pull request by nullkarma**

### CODE REVIEW

File: stackstorm-prometheus/actions/alert_aggregation.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+from st2actions.runners.pythonrunner import Action
+from collections import defaultdict
+
+
+class AlertAggretation(Action):
```
Comment (LindsayHill) [inline stackstorm-prometheus/actions/alert_aggregation.py]: Looks like this should be `AlertAggregation`?

**Comment by nullkarma** [comment]
Both done.

**Comment by LindsayHill** [comment]
Thanks, will get this transferred to new repo.

**Comment by LindsayHill** [comment]
New repo - github.com/StackStorm-Exchange/stackstorm-prometheus

**Pull request closed without merging**